### DISCUSSION TITLE
Pre-release for STIX backend

### DISCUSSION THREAD

**Discussion by barvhaim** [SigmaHQ/pySigma#92] [Ideas]
Hi everyone, I've migrated to old STIX backend from Sigmac to the new pySigma by building it from scratch, so far I've compare the rules between old and new translators and it looks good.

Github https://github.com/barvhaim/pySigma-backend-stix
PyPi https://pypi.org/project/pysigma-backend-stix/

 looking for your feedbacks

**Comment by thomaspatzke** [comment]
Great! I will add it to the [pySigma plugin directory](https://github.com/SigmaHQ/pySigma-plugin-directory), which will integrate it as installable plugin into the next release of the CLI.
